src/screens/LoadWallet.js

```diff
@@ -91,8 +91,7 @@ class LoadWallet extends React.Component {
     // Getting redux variables before cleaning all data
     const { pin, password } = this.props;
     // First we clean what can still be there of a last wallet
-    wallet.cleanWallet();
-    wallet.generateWallet(this.state.words, '', pin, password, true);
+    wallet.generateWallet(this.state.words, '', pin, password, this.props.history);
```

Comment:
You changed a bool parameter (previously `true`) to a list of transactions (`this.props.history`)?

Reply:
Yes. The previous method used to receive `loadHistory` as the last parameter, which was responsible to load/not load the data from the full node when starting the wallet. This is not needed anymore and now I need this `routerHistory` (not a list of transactions) to handle the notification in case of a new tx event.

Reply:
Ok. I saw this change later in the changes. Thanks for the explanation.